### ISSUE TITLE
Organization of Powers/Tags/Traits

### ISSUE THREAD

**Issue by fougerec** [jalensailin/mvrpg#8]
It would be awesome as a QOL thing if.

- powers could be grouped together based on what category they come under
- within the categories if things were listed alphabetically or if they could be dragged into order.

**Comment by jalensailin** [comment]
I agree that this would be a good QOL enhancement. I think that your first request is reasonable for initial release, but the 2nd bullet point would have to come later; for now I'll just go with alphabetic ordering.

**Comment by jalensailin** [comment]
What do you think is a good way to display Powers that are part of multiple power sets?

**Comment by fougerec** [comment]
Maybe a tag system that could also be used for filtering?

**Comment by jalensailin** [comment]
this is definitely a possibility but may push it out of the scope of initial release. ill give it some more thought if there is a simple way to organize things that fit into multiple power categories. one idea is that I simply could show them twice, one in each category, but this may also add bloat to the sheet, especially for higher level characters with a ton of powers.

**Comment by jalensailin** [comment]
I don't yet have a good solution for this, and in my opinion, it shouldn't be a blocker for initial release (even though I originally marked it as such). To be clear, this doesn't mean I am closing the issue or don't find it to be a good enhancement, just that it shouldn't block an initial release. Hope you understand!

**Comment by fougerec** [comment]
For sure.  It's certainly no barrier to being able to play and use the sheet :)

**Comment by 9littlebees** [comment]
Just wanted to add, I think this is sorely needed, especially for enemies of Rank4 and higher, when the list of Powers becomes huge. My recommendation would be to split / filter Powers by Duration (Permanent, Standard, Reaction, Concentration) since in the heat of the moment, that's what you need to see, and it would help to break them up.
